Tests/LibWeb/AccessibilityBridge/tests/test_interfaces.py

```
@@ -0,0 +1,119 @@
+
+        named = None
+
+        def visit(obj, _depth):
+            nonlocal named
+            if named is not None:
+                return
+            try:
+                if obj.get_name() == "Named section":
+                    named = obj
+            except Exception:
+                pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)